build-test.cmd

```diff
@@ -104,6 +104,13 @@ call %__ProjectDir%\init-tools.cmd
 
 REM =========================================================================================
 REM ===
+REM === Deploy debugger tests
```

Comment:
I see. Seems for windows we restore everything in build tests. Basically that's why I let build-tests.cmd to call setup-debuggertests there - just want to do all the restores together. Also for runtests.cmd, seem it just run the tests, it's a little bit weird if we run the tests and download the tests at the same time. Just like the comments previously. 
But for runtests.sh, seems we do restore the dependency runtimes there. That's why I let runtests.sh call debugger-setup.sh. 
If we can confirm that putting my setup-debuggertests.cmd to runtests.cmd isn't obey the design principle, I am fine to let runtests.cmd call setup-debuggertests.cmd.